chapters/remote/deploy-jinad-on-aws-with-terraform.md

```diff
@@ -0,0 +1,125 @@
+# Deploying JinaD on AWS With Terraform
+
+## Overview
+
+Here is a [terraform module jinad-aws](https://registry.terraform.io/modules/jina-ai/jinad-aws/jina/latest) to deploy `JinaD` on AWS by terraform.
```

Comment:
The Jina AI team has released a [terraform module jinad-aws](https://registry.terraform.io/modules/jina-ai/jinad-aws/jina/latest) to deploy `JinaD` on AWS by terraform.

Reply:
And I should remove the `a` before the [terraform module jinad-aws](https://registry.terraform.io/modules/jina-ai/jinad-aws/jina/latest)

Reply:
Done.